container/Exchange.js

```diff
@@ -49,11 +53,19 @@ export default () => {
 
   return (
     <Container>
-      {loading || !translation ? (
-        <Spinner />
-      ) : (
-        <Translation translationResult={translation} />
-      )}
+      {(() => {
+        let element = null
```

Comment:
Wouldn't it make more sense to just return in the if/else-cases?

Reply:
The arrow function expects something to be returned at the end; I kept getting ESLint errors [https://imgur.com/Z21FWf9](otherwise)